src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -5,9 +5,10 @@
 import sys
 from functools import lru_cache
 from pathlib import Path
-from typing import Any, Dict, List, Union
+from typing import Dict, List, Tuple, Union
 
 from sphinx.application import Sphinx
+from sphinx.config import Config
```

Comment:
Let's do
```suggestion
from sphinx.config import Config as SphinxConfig
```
so that the code below is more readable.